ffwd-reporter/src/main/java/com/spotify/metrics/ffwd/FastForwardReporter.java

```diff
@@ -345,6 +365,13 @@ private void reportMetered(final Metric m, Metered value) {
         send(r.attribute("stat", "5m").value(value.getFiveMinuteRate()));
     }
 
+    private void reportDistribution(final com.spotify.ffwd.v1.Metric metric,
+                                    final Distribution distribution) {
+        ByteBuffer byteBuffer = distribution.getValueAndFlush();
+        Value value = Value.distributionValue(byteBuffer);
```

Comment:
I just noticed this PR: https://github.com/spotify/ffwd-client-java/pull/10
Can we update the ffwd API to use ByteString instead of ByteBuffer?
I think it should be safe to change that if no one is using the distribution types yet (since they are not implemented yet)

Reply:
If you look at the entire [PR](spotify/ffwd-client-java#10) you will see that ByteBuffer is transformed into ByteString before serialization.  Moving byteString up in the stack is something we can do in the next PR.